agent/csec_metadata.c

```diff
@@ -57,15 +60,28 @@ int nr_php_csec_get_metadata(const nr_php_csec_metadata_key_t key, char** p) {
     case NR_PHP_CSEC_METADATA_PLICENSE:
       value = NRPRG(app)->plicense;
       break;
+    case NR_PHP_CSEC_METADATA_TRACE_ID:
+      id_value = nr_txn_get_current_trace_id(NRPRG(txn));
+      is_allocated = true;
+      break;
+    case NR_PHP_CSEC_METADATA_SPAN_ID:
+      id_value = nr_txn_get_current_span_id(NRPRG(txn));
```

Comment:
The span id will be different depending on which agent's instrumentation is called first with OAPI